tools/inference/openvino_inference.py

```diff
@@ -4,16 +4,145 @@
 file system, and show the visualization results.
 """
 
-# Copyright (C) 2022 Intel Corporation
+# Copyright (C) 2023 Intel Corporation
```

Comment:
should we do something like this
```suggestion
# Copyright (C) 2022-2023 Intel Corporation
```

Reply:
Actually, I am not sure about the convention here. I am not even sure if people generally increment the number as well.